### PR TITLE
LotW ADIF includes an unexpected tag at the end. This is now properly handled.

### PR DESCRIPTION
**Pull request by sd**

### CODE REVIEW

File: test/adif-parser.test.ts [hunk 1]
```diff
@@ -119,6 +119,17 @@ describe('AdifParser', () => {
...
+
+  it('can parse an ADI from Logbook of the World', () => {
+    const fileContent = fs.readFileSync('test/data/lotw.adi', {
+      encoding: 'ascii',
+    });
+    const expected = fs.readFileSync('test/data/lotw.json', {
+      encoding: 'utf8',
+    });
+    console.log(JSON.stringify(AdifParser.parseAdi(fileContent)));
+    //expect(AdifParser.parseAdi(fileContent)).toEqual(JSON.parse(expected));
```
Comment (xylo04) [inline test/adif-parser.test.ts:131]: What's up with the commented-out expectation here?

**Review by xylo04 — CHANGES_REQUESTED** [review]
Oh yes, I've encountered this before on parsers for other languages. It's very frustrating that LotW includes this non-conformant tag.

**Comment by sd** [comment]
let me fix that commented line and resubmit

**Review by xylo04 — APPROVED** [review]
Thank you!

**Pull request merged**